source/modules/tabs/gestion.rst

```diff
@@ -0,0 +1,78 @@
+* Account net value: this is the automatic calculation of the gross value of a piece of equipment minus the amount of depreciation.
+* Type of depreciation: choice of the type of depreciation between linear and declining balance.
+* Depreciation period: depreciation period expressed in years.
+* Depreciation coefficient: coefficient applied to a straight-line depreciation type in order to obtain the values of the declining balance type. It is therefore only used if the type of depreciation is declining balance.
+* TCO (value+amount of interventions): the total cost of ownership which includes all the constituent elements of an invoiced product.
+* Budget: the budget on which this equipment was purchased See :doc:`Managing the budget <../gestion/budgets>`.
+* Order date: date on which the material was ordered.
+* Purchase date: date on which the equipment was purchased.
+* Delivery date: date on which the equipment was delivered.
+* Startup date: date on which the equipment was put into service.
+* Date of last physical inventory: date of the last physical inventory of the equipment.
+* Comments.
+* Monthly TCO: TCO divided by the number of months between today's date and the date of purchase of the equipment.
+
+Warranty information
+^^^^^^^^^^^^^^^^^^^^
+
+* Warranty start date: date on which the warranty of the equipment starts
+* Warranty information: text qualifying the warranty
+* Warranty period: duration of the warranty expressed in months
+   If a warranty start date and a warranty period are set, the information "Expires on" will appear with a date in red if it is earlier than the current date
+
+All the dates defined can be managed automatically according to changes in the status of the equipment. Some dates can also be copied from another date. All this configuration is done by [entity](administration_entity_delegation.dita).
+
+Tips
+^^^^
+
+GLPI allows you to configure a notification on the expiry of the hardware warranty. This is configurable by entity in :ref:`notification management <notification management>` to define the models and recipients used and in :ref:`the administration of entities <administration-entities>` to enable or disable this feature, define the default values and anticipate the sending of the notification if necessary.
+
+GLPI can perform a simple net book value calculation based on straight-line or declining balance depreciation. To do so, a certain amount of information (value, date, etc.) must be entered.
+The user must also enter the date of the financial year in :ref:`the general configuration <configuration-generale>` and the date of the financial year in :ref:`the general configuration <configuration-generale>`. For more information on the notions of depreciation, you can refer to `the Wikipedia article on accounting depreciation <https://fr.wikipedia.org/wiki/Amortissement_comptable#Modes_d.E2.80.99amortissement_accept.C3.A9s_comptablement>`_.
```

Comment:
We should probably directly include the relevant information from this Wikipedia page, or find a localized resource. I couldn't find a direct replacement for this page in English.

Reply:
Well, as far as I've seen, the closest english wikipedia resource would be:
https://en.wikipedia.org/wiki/Amortization_schedule#Methods_of_amortization

But maybe should we remove it; I'm not sure one can find its accounting rules on wikipedia for every country (I'm not even sure what GLPI proposes can be used for any other country)